app/controllers/agenda/documents.js

```diff
@@ -126,12 +127,7 @@ export default class AgendaDocumentsController extends Controller {
 
   @action
   saveBatchDetails() {
-    this.refresh();
+    this.router.refresh();
     this.isOpenBatchDetailsModal = false;
   }
-
-  @action
-  refresh() {
-    this.send('reloadModel');
-  }
 }
```

Comment:
What @erikap said.
The `refresh` action in the controller has been removed but it still used in the template.
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/KAS-3809-replace-send-by-router-refresh/app/templates/agenda/documents.hbs#L34
Perhaps it made more sense to keep the action with `this.router.refresh()` and use `this.refresh()` in the 2 methods in this controller that need to trigger a refresh.

There are more cases.
Perhaps best to check all templates of the controllers where the action was removed.